src/libcommon/log/sentry/sentryhandler.h

```diff
@@ -136,5 +139,8 @@ class SentryHandler {
         std::unordered_map<std::string, SentryEvent, StringHash, std::equal_to<>> _events;
         SentryConfidentialityLevel _globalConfidentialityLevel = SentryConfidentialityLevel::Anonymous; // Default value
         SentryConfidentialityLevel _lastConfidentialityLevel = SentryConfidentialityLevel::None;
+        const int _sentryMaxCaptureCountBeforeRateLimit = 10; // Number of capture before rate limiting an event
```

Comment:
`static` class members ?

Reply:
Now I understand that these constants are changed in the derived class for unit testing.